### DISCUSSION TITLE
UTM SE install in AltStore can be crashed

### DISCUSSION THREAD

**Discussion by JYN2018X** [utmapp/UTM#5862] [UTM Help (iOS)]
**Describe the issue**
I installed UTM SE in Altstore but it's crashed

**Configuration (required)**
* UTM Version: 4.2.5
* OS Version: iOS 15.7.5
* Device Model: iPhone 7
* Unjailbreaken Device
* Install UTM in AltStore

**Crash log**
Install UTM SE in AltStore is crashed

**Debug log**
For all issues, _including_ crashes, you should attach a debug log. Open UTM, and open the settings for the VM you wish to launch. Near the bottom of the configuration options (or at the top of the `QEMU` page) is `Debug Log`. Turn it on and save the VM. After you experience the issue, quit the VM and re-launch UTM. Open the VM settings again and select `Export Log...` and attach it here.

**Upload VM**
(Optional) If possible, upload the `config.plist` inside the `.utm`. If you do not have this, you can upload the entire `.utm` but note this contains your personal data. Since GitHub has an attachment size limit, you may want to upload to another service such as Google Drive. Link it here.

**Comment by osy** [comment]
Please attach a crash log.

**Comment by bradycodemaster** [comment]
you cant if the crash log is in utm and utm cant be instaled osy

**Comment by minkbob** [comment]
I had the same thing happen on an iPad Air 2 with iOS 15.8. I attempted to install UTM SE several times and had AltStore crash on most of my attempts, but had one attempt fail with an error message:
Install UTM SE Failed
NSCocoaErrorDomain 512
UTM SE could not be downloaded. The file "emu-riscv32-softmmu" couldn't be saved in the folder "emu-riscv32-softmmu.framework".

**Comment by osy** [comment]
AltStore issues are not handled here. Please follow up with AltStore devs.

Sometimes install issues are due to lack of free space on the device.
